addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
there is no d3d9 devices any longer in any case. it's a d3d11 device now. and this variable is just the same device. it's just passed once instead of every time start is called. i found it to be a waste.